hawkular-wildfly-agent/src/main/resources/schema/hawkular-agent-monitor-subsystem.xsd

```diff
@@ -360,4 +363,18 @@
     <xs:attribute name="time-units" type="timeUnitsType"/>
   </xs:complexType>
 
+  <xs:complexType name="operationParamDmr">
+    <xs:attribute name="name"         type="xs:string" use="required"/>
+    <xs:attribute name="type"         type="paramTypeType" use="optional" default="string"/>
```

Comment:
following the standard in this .xsd, so not explicitly specify "use="optional"" since that is the default - makes the file noisy otherwise.